pkg/api/v1beta1/types.go

```diff
@@ -122,6 +122,8 @@ type PersistentVolumeSource struct {
 	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"persistentDisk" description:"GCE disk resource provisioned by an admin"`
+	// NFS represents an NFS mount on the host that persists beyond a pod's lifetime
+	NFS *NFSVolumeSource `json:"nfs" description:"NFS volume that will be mounted in the host machine "`
```

Comment:
Do you mind if I leave them in?  I added them so I could test the indexing of different access modes on volumes.  My tests create GCE and NFS example volumes for indexing, sorting, and searching.

I can change it to HostPath, if you prefer, to similarly test indexing by access modes.  I thought NFS made for a better test.